frodo.js

```diff
@@ -1,81 +1,86 @@
 /**
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?